core/src/main/resources/struts-beans.xml

```diff
@@ -166,6 +166,8 @@
           class="com.opensymphony.xwork2.validator.DefaultValidatorFileParser"/>
 
     <bean class="com.opensymphony.xwork2.ognl.OgnlUtil"/>
+    <bean type="com.opensymphony.xwork2.ognl.OgnlGuard" name="struts"
+          class="com.opensymphony.xwork2.ognl.StrutsOgnlGuard" />
```

Comment:
Just missed that, I prefer to use `org.apache.struts2.ognl` namespace as `com.opensymphony.xwork2` should be removed at some point (in 7.x)